app/services/learning_progress_service.py

```
@@ -0,0 +1,14 @@
+from typing import List
```

Ruff (0.8.2)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)